pgcontents/pgmanager.py

```diff
@@ -79,15 +73,13 @@ def _checkpoints_class_default(self):
         return PostgresCheckpoints
 
     def _checkpoints_kwargs_default(self):
-        kw = super(PostgresContentsManager, self)._checkpoints_kwargs_default()
-        kw.update(
-            {
-                'db_url': self.db_url,
-                'user_id': self.user_id,
-                'create_user_on_startup': self.create_user_on_startup,
-            }
-        )
-        return kw
+        return {
+            'db_url': self.db_url,
+            'user_id': self.user_id,
+            'create_user_on_startup': self.create_user_on_startup,
+            'crypto': self.crypto,
+            'max_file_size_bytes': self.max_file_size_bytes,
+        }
```

Comment:
Is there a reason you stopped getting the default kwargs from the super class as you were doing before? I understand the need to add the two new kwargs, just don't follow why you no longer get the default kwargs from the super class.